src/Confluent.RestClient/ConfluentClient.cs

```diff
@@ -193,6 +186,26 @@ private HttpRequestMessage CreateRequestMessage(HttpMethod method, string reques
             }
         }
 
+        private async Task<HttpResponseMessage> SendRequest(HttpRequestMessage request)
+        {
+            HttpResponseMessage response;
+            try
+            {
+                response = await _client.SendAsync(request);
```

Comment:
we can just return here as we throw exception on catch, `return await _client.SendAsync(request);` 

Don't need that variable either